client/apps/Admin/src/pages/HourReportPage.tsx

```
@@ -0,0 +1,299 @@
+
+                <div className="hour-report-page__pagination">
+                    <button
+                        onClick={() => table.firstPage()}
+                        disabled={!table.getCanPreviousPage()}
+                        className="hour-report-page__pagination-btn"
+                    >
+                        ‹‹
+                    </button>
+                    <button
+                        onClick={() => table.previousPage()}
+                        disabled={!table.getCanPreviousPage()}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)